dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DialogueException.java

```diff
@@ -38,4 +38,24 @@ public String getLogMessage() {
     public List<Arg<?>> getArgs() {
         return ImmutableList.of();
     }
+
+    private static String copyWhitelistedSafeMessages(Throwable cause) {
+        if (cause == null) {
+            return MESSAGE;
+        }
+
+        String causeMessage = cause.getMessage();
+        if (causeMessage == null) {
+            return MESSAGE;
+        }
+
+        switch (causeMessage) {
+            case "Connection reset":
+            case "Connection reset by peer":
+            case "Broken pipe (Write failed)":
```

Comment:
I wonder if some of these should live in sls-logging instead. It would get a bit weird to support, but they would always "just work".